app/src/filters/index.js

```diff
@@ -45,7 +45,7 @@ export const formatAddress = (address, length = 4) => {
 }
 
 export const resolveValidatorName = (address, validators) => {
-  if (validators[address]) {
+  if (validators[address] && validators[address].name.length < 20) {
```

Comment:
why is this needed?

Reply:
Because names in Substrate can be the operatorAddress, being super long and thus breaking the extension styles

Reply:
alright! please leave comment. this is not at all obvious

Reply:
oh true! Good catch. I thought I had